### DISCUSSION TITLE
Using custom containers inside tabs

### DISCUSSION THREAD

**Discussion by YconquestY** [vuepress-theme-hope/vuepress-theme-hope#2157] [Q&A] [answered]
I saw in the [doc](https://vuepress-theme-hope.github.io/v2/guide/markdown/tabs.html) that we may use tabs in custom containers. However, when I do the reverse, i.e., containers in tabs, it seems that the theme will identify the boundary **incorrectly**. Below is an example:
```markdown
::: tabs

@tab foo

::: info
Thie is a container.
:::

@tab boo

:::
```

The theme treats the ending `:::` of the `info` container as an end of tabs. May I know how to resolve such a conflict?

**Comment by Mister-Hope** [comment, marked as answer]
```md
:::: tabs

@tab foo

::: info
Thie is a container.
:::

@tab boo

::::
```
